source/linux/Overview/Processor_SDK_Linux_create_SD_card.rst

```diff
@@ -130,15 +133,15 @@ Create SD Card using bmap-tools
          Then generate a bmap file from the decompressed WIC image with the following command.
          This step can be skipped but the bmap file significantly reduces the time taken to flash the SD card.
 
-         .. code-block:: console
+         .. parsed-literal::
 
-             bmaptool create -o tisdk-default-image.bmap tisdk-default-image-<machine>.wic
+             bmaptool create -o tisdk-|__IMAGE_TYPE__|-image.bmap tisdk-|__IMAGE_TYPE__|-image-<machine>.wic
 
       c) Then write the WIC image to the SD card with the following command:
 
-         .. code-block:: console
+         .. parsed-literal::
 
-             sudo bmaptool copy --bmap tisdk-default-image.bmap tisdk-default-image-<machine>.wic /dev/sdx
+             sudo bmaptool copy --bmap tisdk-|__IMAGE_TYPE__|-image.bmap tisdk-|__IMAGE_TYPE__|-image-<machine>.wic /dev/sdx
```

Comment:
If you're going to make this a parsed literal, might as well replace `<machine>` with `|__SDK_BUILD_MACHINE__|`

Reply:
Updated it